ceph-perf-pull-requests/config/definitions/ceph-perf-pull-requests.yml

```diff
@@ -166,12 +166,12 @@
           . /etc/os-release || ID=ubuntu
           case $ID in
           debian|ubuntu)
-              sudo env DEBIAN_FRONTEND=noninteractive apt-get install -y python3-yaml python3-lxml python3-prettytable clang-10
+              sudo env DEBIAN_FRONTEND=noninteractive apt-get install -y python3-yaml python3-lxml python3-prettytable clang-12
               ;;
           centos|rhel)
-              sudo dnf module disable -y llvm-toolset
-              sudo dnf copr enable -y tchaikov/llvm-toolset-10 centos-stream-8-x86_64
```

Comment:
Do we maybe want to make sure these are both disabled?

Reply:
good point. the repo is always removed. we can even drop that line after, probably 2 months, assuming it's long enough to get rid of the copr repo on every jenkins node.